testing/rspamd/APKBUILD

```diff
@@ -35,6 +35,7 @@ build() {
 		-DRSPAMD_USER=$pkgusers \
 		-DRSPAMD_GROUP=$pkggroups \
 		-DENABLE_HIREDIS=ON \
+		-DENABLE_LUAJIT=OFF \
```

Comment:
Maybe not on Alpine. :) https://github.com/alpinelinux/aports/commit/8bdc98d68351be28bf94e42b1fc6aa62cfe7a206#diff-0faa079b2a5bf9f91d71fe9aef8837f6